code/style.css

```diff
@@ -1,7 +1,327 @@
 /* the + sign is used to select the sibling that comes immediately next to the element you selected.
 in this case it is the div with the answer class next to the div with the active class.
 this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
+body, html {
+  margin:0px;
+  font-family: 'Cormorant', serif;
+  color: black;
+  box-sizing: border-box;
+}
+
+/*Header*/
+
+.header-section {
+  position:relative;
+  width: 100%;
+  height: 70vh;
+  display: flex;
+  align-items: flex-start;
+  justify-content: center;
+  text-align: center;
+  object-fit: cover;
+  overflow:hidden;
+  margin:0px;
+  padding:0px;
+}
+
+.text-header{
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content:center;
+  margin-top: 75px;
+  border:0px;
+}
+
+/*Video*/
+
+.hero-video {
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+  z-index: -1; 
+  margin:0px;
+}
+/*Text*/
+
+h1{
+  font-size: 30px;
+  text-align: center; 
+}
+
+h2{
+  font-size: 20px;
+}
+
+h3{
+ font-size: 22px; 
+}
+
+p{
+  font-size: 20px;
+  font-weight: bold;
+}
+
+/*Button*/
+
+  button {
+  background-color: lightgrey;
+  opacity:0.7;
+  border-color: 10px black;
+  border-radius: 10px;
+  color: black;
+  font-family: 'Cormorant', serif; 
+  padding: 10px 32px;
+  text-align: center;
+  text-decoration: none;
+  display:block;
+  font-size: 20px;
+  margin: 2px 2px;
+  cursor: pointer;
+}
+
+/*Wrapper form and faq*/
+
+.form-wrapper{
+  display:flex;
+  flex-wrap: wrap;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center; 
+}
+/*Form*/
+
+.form{
+  display: flex;
+  position: relative;
+  margin:20px 0px 20px 0px;
+  padding:10px;
+  font-size: 20px;
+  font-weight: bold;
+  width:80vw;
+  background-color:rgb(255, 208, 254,0.4); 
+  border: 1px solid rgb(00, 00, 00,0.4);
+}
+
+ input{
+  display: block;
+}
+
+input[type=checkbox] {
+  position: relative;
+  display: inline-block;
+  margin:0px 0px 25px 0px;
+}
+
+input[type=radio] {
+  
+  display: inline-block;
+  margin:0px 0px 25px 0px;
+}
+/*Accordion*/
+
+.accordion{
```

Comment:
The accordion container expands both up and down when you open the accordion, it would make more sense if it only expand downwards, maybe something to do with overflow?